### ISSUE TITLE
the loss of discriminator

### ISSUE THREAD

**Issue by YoungLer** [zelaki/eqvae#9]
Hi,
thank for the great work!

I have a question to ask:
Is the loss of discriminator stable during the process of fine-tuning VAE? Does logits-real, logits_fake, and disc_loss meet your expectations? I am trying an idea similar to yours for this job, but I have encountered some problems. Your reply can help me a lot!
Thank you for your excellent work~
